proposal.h

```diff
@@ -0,0 +1,14 @@
+/* one */
+proto:
+int nrf_setdnsaddr(const char *dns, size_t len);
+usage:
+nrf_setdnsaddr("8.8.8.8", strlen("8.8.8.8)"));
+
+/* two */
+proto:
+int nrf_setdnsaddr(int family, void *in_addr);
```

Comment:
I was thinking about using `sockaddr` pointer, which has the `family` information within, and can hold both, IPv4 and IPv6. But I see that there's a binary incompability betweent Zephyr/nRF sockets, so perhaps it was not a good idea.